src/testdata/configdetection/compose.yaml

```
@@ -0,0 +1,10 @@
+services:
+  configdetection: 
+    image: alpine
+    environment:
+      - API_KEY=50m34p1k3y # keyword detector
+      - AWS_CLIENT_ID=AROA1234567890ABCDEF # aws_client_id detector
+      - GH_PAT=ghp_aBcDeFgHiJkLmNoPqRsTuVwXyZ1234567890 # github detector
+      - HIGH_ENTROPY_STRING=1234567890abcdef # high_entropy_string detector
+      - MY_URL=https://user:p455w0rd@example.com # url_password detector
+      - NOT_A_CONFIG=notaconfigvalue
```

nit, this could still be "config" if the user wanted, but it's here because it's not "sensitive":
```suggestion
      - NOT_A_SECRET=notasecretvalue
```